<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LMTDJO9" failures="2" tests="3" name="Test" time="2.697" errors="0" timestamp="2024-02-05T19:07:45 EET">
  <testcase classname="tests.HappyScenario" name="virtualUser" time="0.351">
    <failure type="java.lang.AssertionError" message="User Logged in successfully expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: User Logged in successfully expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at pages.LoginPage.checkSuccess(LoginPage.java:67)
at tests.HappyScenario.virtualUser(HappyScenario.java:65)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- virtualUser -->
  <testcase classname="tests.HappyScenario" name="successfulFlow" time="0.343">
    <failure type="java.lang.AssertionError" message="User Logged in successfully expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: User Logged in successfully expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at pages.LoginPage.checkSuccess(LoginPage.java:67)
at tests.HappyScenario.successfulFlow(HappyScenario.java:40)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- successfulFlow -->
  <testcase classname="tests.HappyScenario" name="lockedUser" time="0.353"/>
</testsuite> <!-- Test -->
